### PR TITLE
Add api routes to list all teams, projects and runs

### LINKED ISSUE #219: Add routes to the API to get lists of entities
We need more routes in the API:
- GET /teams/organization/{organization_id} to list all teams of a given organization.
- GET /projects/team/{team_id} to list all projects of a given team.
- GET /runs/experiment/{experiment_id} to list all runs of a given experiment.

### PR DESCRIPTION
**Pull request by Minervabooks**
This change adds new routes in the API.

Closes #219 

-     GET /teams/organization/{organization_id} to list all teams of a given organization.
-     GET /projects/team/{team_id} to list all projects of a given team.
-     GET /runs/experiment/{experiment_id} to list all runs of a given experiment.

### CODE REVIEW

**Review by SaboniAmine — APPROVED** [review]
LGTM !
Could you please update the postman collection in addition to the tests you wrote ?
You can focus on the important ones and drop the meaningless request examples already present in the collection.

**Comment by Minervabooks** [comment]
@SaboniAmine Postman collection updated, can you please check if this is okay?

File: carbonserver/carbonserver/api/infra/repositories/repository_projects.py [hunk 1]
```diff
@@ -36,6 +37,26 @@ def get_one_project(self, project_id):
...
+        :rtype: List[schemas.Project]
+        """
+        with self.session_factory() as session:
+            res = session.query(SqlModelProject).filter(
+                SqlModelProject.team_id == team_id
+            )
+            if res.first() is None:
+                return []
+            else:
+                projects = []
```
Comment (benoit-cty) [inline carbonserver/carbonserver/api/infra/repositories/repository_projects.py:54]: You could have write it in one line:
return [self.map_sql_to_schema(e) for e in res]

**Review by benoit-cty — APPROVED** [review]
Thanks for your contribution

**Pull request merged**